src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -195,21 +263,41 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         stream.print(getStandardUsagePreamble(callerArguments.getClass()) + getUsagePreamble());
         stream.println("\n" + getVersion());
 
-        // filter on common and partition on optional
-        final Map<Boolean, List<ArgumentDefinition>> argMap = argumentDefinitions.stream()
+        // filter on common and partition on plugin-controlled
+        final Map<Boolean, List<ArgumentDefinition>> allArgsMap = argumentDefinitions.stream()
                 .filter(argumentDefinition -> printCommon || !argumentDefinition.isCommon)
-                .collect(Collectors.partitioningBy(a -> a.optional));
+                .collect(Collectors.partitioningBy(a -> a.controllingDescriptor == null));
+
+        List<ArgumentDefinition> nonPluginArgs = allArgsMap.get(true);
+        if (null != nonPluginArgs && !nonPluginArgs.isEmpty()) {
+            // partition the non-plugin args on optional
+            final Map<Boolean, List<ArgumentDefinition>> unconditionalArgsMap = nonPluginArgs.stream()
+                    .collect(Collectors.partitioningBy(a -> a.optional));
+
+            final List<ArgumentDefinition> reqArgs = unconditionalArgsMap.get(false); // required args
+            if (reqArgs != null && !reqArgs.isEmpty()) {
+                stream.println("\n\nRequired Arguments:\n");
+                reqArgs.stream().forEach(argumentDefinition -> printArgumentUsage(stream, argumentDefinition));
+            }
 
-        final List<ArgumentDefinition> reqArgs = argMap.get(false); // required args
-        if (reqArgs != null && !reqArgs.isEmpty()) {
-            stream.println("\n\nRequired Arguments:\n");
-            reqArgs.stream().forEach(argumentDefinition -> printArgumentUsage(stream, argumentDefinition));
+            final List<ArgumentDefinition> optArgs = unconditionalArgsMap.get(true); // optional args
+            if (optArgs != null && !optArgs.isEmpty()) {
+                stream.println("\nOptional Arguments:\n");
+                optArgs.stream().forEach(argumentDefinition -> printArgumentUsage(stream, argumentDefinition));
+            }
         }
 
-        final List<ArgumentDefinition> optArgs = argMap.get(true); // optional args
-        if (optArgs != null && !optArgs.isEmpty()) {
-            stream.println("\nOptional Arguments:\n");
-            optArgs.stream().forEach(argumentDefinition -> printArgumentUsage(stream, argumentDefinition));
+        // now the contingent/dependent args (those controlled by a plugin descriptor)
+        List<ArgumentDefinition> conditionalArgs = allArgsMap.get(false);
+        if (null != conditionalArgs && !conditionalArgs.isEmpty()) {
+            // group the argdefs by their parent (plugin) class
+            Map<String, List<ArgumentDefinition>> groupedArgsMap = conditionalArgs.stream()
+                    .collect(Collectors.groupingBy(argDef -> argDef.parent.getClass().getSimpleName()));
+            stream.println("\nConditional Arguments:\n");
```

Comment:
This is nitpicky, but it would be nice if instead of just "Conditional Arguments" these were grouped by plugin descriptor.  I.e. All the read filters are together in a block that says "Read Filter Arguments", and a second plugin would be listed separately.  We could patch that in a later pull request if you want though.